### PR TITLE
pw-loopback: refresh

### PR DESCRIPTION
**Pull request by Managor**
- [x] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**
0.3.39

### CODE REVIEW

File: pages/linux/pw-loopback.md [hunk 1]
```diff
@@ -7,14 +7,22 @@
...
-`pw-loopback --capture-props='{{media.class=Audio/Sink node.name=my-sink audio.position=[FL FR]}}' --playback-props='{{audio.position=[FR FL]}}'`
+`pw-loopback -m '{{[FL FR]}}' --playback-props='{{media.class=Audio/Source}}'`
 
-- Create a source that can be selected in applications that swaps the left and right channels from another source:
+- Create a dummy loopback device that doesn't automatically connect to anything:
 
-`pw-loopback --capture-props='{{audio.position=[FR FL]}}' --playback-props='{{media.class=Audio/Source node.name=my-source audio.position=[FL FR]}}'`
+`pw-loopback -m '{{[FL FR]}}' --capture-props='{{media.class=Audio/Sink}}' --playback-props='{{media.class=Audio/Source}}'`
+
+- Create a loopback device that automatically connects to the speakers and swaps the left and right channels between the sink and source:
```
Comment (marchersimon) [inline pages/linux/pw-loopback.md:22]: I don't quite understand the `swaps the left and right channels between the sink and source`. Is there a way to explain this a bit without making it longer?

File: pages/linux/pw-loopback.md [hunk 2]
```diff
@@ -7,14 +7,22 @@
...
+
+`pw-loopback --capture-props='{{media.class=Audio/Sink audio.position=[FL FR]}}' --playback-props='{{audio.position=[FR FL]}}'`
+
+- Create a loopback device that automatically connects to the microphone and swaps the left and right channels between the sink and source:
```
Comment (marchersimon) [inline pages/linux/pw-loopback.md:26]: Same here

File: pages/linux/pw-loopback.md (hunk 1 above)
Comment (Managor) [reply pages/linux/pw-loopback.md:22]: I think that's the correct terminology. The audio enters the sink and exits out of the audio source with the channels stereo channels swapped

Comment (sbrl) [reply pages/linux/pw-loopback.md:22]: Hmmm, I'm not sure there's a better way we can word this. Thanks for the clarification here @Managor - I think audio stuff can just be complicated sometimes.

What about adding a 2nd description line something like this?

```markdown
> A sink is ....., and a source is ......
```

Comment (marchersimon) [reply pages/linux/pw-loopback.md:22]: Well, after the explanation it makes sense. I assume if someone is working with pipewire it shouldn't be a problem.
@sbrl isn't `sink`/`source` a rather common terminology?

**Review by marchersimon — APPROVED** [review]
Thanks @Managor

File: pages/linux/pw-loopback.md (hunk 1 above)
Comment (Managor) [reply pages/linux/pw-loopback.md:22]: If you check the More information site of this page, sink and source are used there extensively

Comment (sbrl) [reply pages/linux/pw-loopback.md:22]: Yeah, they are @marchersimon - was just pointing out something that could potentially improve clarity

At any rate, I'm not sure we can really improve on the example description here

**Review by sbrl — APPROVED** [review]
Thanks, @Managor!

**Pull request merged**